L1Trigger/CSCTrackFinder/BuildFile.xml

```diff
@@ -12,6 +12,7 @@
 <use   name="DataFormats/L1DTTrackFinder"/>
 <use   name="DataFormats/L1GlobalMuonTrigger"/>
 <use   name="boost"/>
+<use   name="CSCTrackFinderEmulation"/>
 
 <flags ADD_SUBDIR="1"/>
```

Comment:
@degano , you can remove ADD_SUBDIR flag too. It was there to instruct scram to look for source files in subdirs of src. As there are no sub-directories any more, so no need to have this flag.